src/e2e/scala/temple/SimpleE2ETest.scala

```diff
@@ -43,5 +43,65 @@ class SimpleE2ETest extends FlatSpec with Matchers {
     // The content of the templeuser/Dockerfile file should be correct
     val templeUserDockerfile = Files.readString(basePath.resolve("templeuser").resolve("Dockerfile"))
     templeUserDockerfile shouldBe SimpleE2ETestData.dockerfile
+
+    // Only one file should be present in the kong folder
+    val expectedKongFiles = Set("configure-kong.sh").map(dir => basePath.resolve("kong").resolve(dir))
+    Files.list(basePath.resolve("kong")).toScala(Set) shouldBe expectedKongFiles
+
+    // The content of the kong/configure-kong.sh file should be correct
+    val templeKongConf = Files.readString(basePath.resolve("kong").resolve("configure-kong.sh"))
+    templeKongConf shouldBe SimpleE2ETestData.configureKong
+
+    // Only these files should be present in the kube/kong folder
+    val expectedKongKubeFiles = Set(
+      "kong-deployment.yaml",
+      "kong-service.yaml",
+      "kong-db-deployment.yaml",
+      "kong-db-service.yaml",
+      "kong-migration-job.yaml",
+    ).map(dir => basePath.resolve("kube/kong").resolve(dir))
+    Files.list(basePath.resolve("kube/kong")).toScala(Set) shouldBe expectedKongKubeFiles
+
+    // The content of the kube/kong/ files should be correct
+    val templeKongKubeDeployment = Files.readString(basePath.resolve("kube/kong").resolve("kong-deployment.yaml"))
+    templeKongKubeDeployment shouldBe FileUtils.readResources("kube/kong/kong-deployment.yaml")
+
+    val templeKongKubeDbDeployment = Files.readString(basePath.resolve("kube/kong").resolve("kong-db-deployment.yaml"))
+    templeKongKubeDbDeployment shouldBe FileUtils.readResources("kube/kong/kong-db-deployment.yaml")
+
+    val templeKongKubeService = Files.readString(basePath.resolve("kube/kong").resolve("kong-service.yaml"))
+    templeKongKubeService shouldBe FileUtils.readResources("kube/kong/kong-service.yaml")
+
+    val templeKongKubeDbService = Files.readString(basePath.resolve("kube/kong").resolve("kong-db-service.yaml"))
+    templeKongKubeDbService shouldBe FileUtils.readResources("kube/kong/kong-db-service.yaml")
+
+    val templeKongKubeMigration = Files.readString(basePath.resolve("kube/kong").resolve("kong-migration-job.yaml"))
+    templeKongKubeMigration shouldBe FileUtils.readResources("kube/kong/kong-migration-job.yaml")
+
+    // Only these files should be present in the kube/TempleUser folder
```

Comment:
We may want to lower case the `TempleUser`, to be consistent with the service folder name?

Reply:
I realise this is quite a few changes 😆, but we should probably be consistent